edb/server/main.py

```diff
@@ -515,6 +515,7 @@ def _protocol_version(
         '--max-protocol', type=str, callback=_protocol_version,
         default='.'.join(map(str, mng_port.CURRENT_PROTOCOL)),
         help='maximum supported and advertized client protocol version'),
+    click.version_option(),
```

Comment:
We need to use `buildmeta.get_version()` here for consistency with `sys::get_version()` and server logs.

Reply:
Fixed. Unfortunately `buildmeta.get_version()` doesn't work at import time, hence PR is a bit more complex.